content/en/user-guide/aws/rds/index.md

```diff
@@ -137,69 +128,152 @@ $ awslocal rds-data execute-statement --database test --resource-arn arn:aws:rds
+You can retrieve the hostname and port of your created instance either from the preceding output or by using the [`DescribeDbInstances`](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_DescribeDBInstances.html) API. Run the following command to connect to the database:
 
+{{< command >}}
 $ PORT=$(awslocal rds describe-db-instances --db-instance-identifier mydb | jq -r ".DBInstances[0].Endpoint.Port")
 $ HOST=$(awslocal rds describe-db-instances --db-instance-identifier mydb | jq -r ".DBInstances[0].Endpoint.Address")
+{{< / command >}}
 
-$ PGPASSWORD=$MASTER_PW psql -d $DB_NAME -U $MASTER_USER -p $PORT -h $HOST -w -c 'CREATE USER myiam WITH LOGIN'
+Next, you can connect to the database using the master username and password:
 
+{{< command >}}
+$ PGPASSWORD=$MASTER_PW psql -d $DB_NAME -U $MASTER_USER -p $PORT -h $HOST -w -c 'CREATE USER myiam WITH LOGIN'
 $ PGPASSWORD=$MASTER_PW psql -d $DB_NAME -U $MASTER_USER -p $PORT -h $HOST -w -c 'GRANT rds_iam TO myiam'
+{{< / command >}}
+
+### Create a token
 
+You can create a token for the user you generated using the [`generate-db-auth-token`](https://docs.aws.amazon.com/cli/latest/reference/rds/generate-db-auth-token.html) command:
+
+{{< command >}}
 $ TOKEN=$(awslocal rds generate-db-auth-token --username myiam --hostname $HOST --port $PORT)
+{{< / command >}}
 
+You can now connect to the database utilizing the user you generated and the token obtained in the previous step as the password:
+
+{{< command >}}
 $ PGPASSWORD=$TOKEN psql -d $DB_NAME -U myiam -w -p $PORT -h $HOST
 {{< / command >}}
 
 ## Global Database Support
 
-LocalStack supports [Aurora Global Database](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-global-database.html), with some limitations:
+LocalStack extends support for [Aurora Global Database](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-global-database.html) with certain limitations:
+
+- Creating a global database will result in the generation of a single local database. All clusters and instances associated with the global database will share a common endpoint.
+- It's important to note that clusters removed from a global database lose their ability to function as standalone clusters, differing from their intended behavior on AWS.
+- At present, the capability for persistence within global databases is not available.
+
+## Resource Browser
+
+The LocalStack Web Application provides a Resource Browser for managing RDS instances and clusters. You can access the Resource Browser by opening the LocalStack Web Application in your browser, navigating to the **Resources** section, and then clicking on **RDS** under the **Database** section.
+
+<img src="rds-resource-browser.png" alt="RDS Resource Browser" title="RDS Resource Browser" width="900" />
```

Comment:
For me the image is not showing in the preview (only showing the "alt" text):
https://localstack-docs-preview-pr-764.surge.sh/user-guide/aws/rds/#resource-browser. In the console it states `Image corrupt or truncated`. 
Could it have something to do with the preview? Because on the same deployment I see this error for the cloudwatch docs as well, but on our official, released docs it works just fine 🤔

Reply:
Works fine on my end

![image](https://github.com/localstack/docs/assets/47351025/40b057ae-55b9-4df3-bc3e-0741fd930977)